core/src/ast/types.rs

```diff
@@ -188,7 +201,7 @@ impl PathType {
                     }
                     Some(ModSymbol::CustomType(t)) => {
                         if i == local_path.elements.len() - 1 {
-                            return (cur_path, t);
+                            return (cur_path, CustomNamedConstruct::CustomType(t.clone()));
```

Comment:
nit: please no cloning, we should have the enum contain references

Reply:
OR just use two different methods for this since you always know in the callsite whether a trait or type was requested, however having an enum here allows for slightly nicer error messages during lowering, so that's nice

Reply:
deprecated b/c we added `PathTrait` for traits instead of `PathType`